contrib/automation_tests/test_cases/main_window.py

```diff
@@ -84,3 +84,37 @@ def _execute(self):
             lambda: self.suite.application.top_window().class_name() ==
             "orbit_session_setup::SessionSetupDialog", 30)
         self.suite.top_window(force_update=True)
+
+
+class DismissDialog(E2ETestCase):
+    """
+    Find a dialog, compare the title and dismiss it by clicking 'OK'.
+    """
+
+    def _execute(self, title_contains: str):
+        searchRegex = ".*" + title_contains + ".*"
+        dialog_box = self.suite.top_window().child_window(title_re=searchRegex,
+                                                          control_type="Window")
+        self.expect_true(dialog_box is not None, 'Dialog found.')
+        ok_button = dialog_box.descendants(control_type='Button', title='OK')[0]
+        self.expect_true(ok_button is not None, 'OK Button found.')
+        ok_button.click_input()
+
+
+class RenameMoveCaptureFile(E2ETestCase):
+    """
+    Click menu entry "Rename/Move Capture File", enter new_capture_path and confirm it.
+    """
+
+    def _execute(self, new_capture_path):
+        app_menu = self.suite.top_window().descendants(control_type="MenuBar")[1]
+        app_menu.item_by_path("File->Rename/Move Capture File").click_input()
+
+        wait_for_condition(lambda: self.find_control('Edit', 'File name:') is not None,
+                           max_seconds=120)
+        file_name_edit = self.find_control('Edit', 'File name:')
+
+        file_name_edit.set_edit_text(new_capture_path)
+
+        save_button = self.find_control('Button', 'Save')
+        save_button.click_input()
```

Comment:
This is probaby missing some sort of wait so that we can guarantee the dialog is closed before we exit this method. Since RenameMoveCaptureFile is the last step in the test this might otherwise swallow the alt+f4 to end Orbit.

Reply:
compare here: https://github.com/google/orbit/pull/4186

Reply:
found a neat solution. ptal

Reply:
This is nice! 
Can you add this as well to orbit_e2e.py, CloseOrbit, just before we send the alt+f4?

Having it in the individual tests is good, but I guess folks will forget about it and having it in CloseOrbit doesn't hurt and probaly fix the entire class of test failures.